src/System.Net.Http/tests/FunctionalTests/CancellationTest.cs

```diff
@@ -0,0 +1,125 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics;
+using System.IO;
+using System.Net.Test.Common;
+using System.Threading;
+using System.Threading.Tasks;
+
+using Xunit;
+using Xunit.Abstractions;
+
+namespace System.Net.Http.Functional.Tests
+{
+    public class CancellationTest
+    {
+        // TODO: Issue #8692. Move this test server capability to Azure test server or Loopback server.
+        private const string FastHeadersSlowBodyHost = "winrtnet.corp.microsoft.com";
```

Comment:
I will remove the DNS name of the internal server and replace it with a TODO. Since the tests are disabled right now w/ ActiveIssue, it will be ok.  There is a GitHub issue tracking to re-enable this.